src/pages/Proposal.vue

```diff
@@ -1,137 +1,142 @@
 <template>
-	<a-row>
-		<a-col :span="12" :offset="6">
-			<div v-if="error" class="margin_bottom_sm">
-				<a-alert
-					message="Error"
-					:description="error"
-					type="error"
-					show-icon
-					closable
-					@close="error = ''"
-				/>
-			</div>
-			<a-form
-				:label-col="{ span: 10 }"
-				:wrapper-col="{ span: 20 }"
-				:model="formState"
-				name="Add Proposal"
-				autocomplete="off"
-				@finish="onFinish"
-				@finishFailed="onFinishFailed"
-				@validate-messages="validateMessages"
-			>
-				<a-form-item
-					label="URL"
-					name="url"
-					:rules="[{ required: true, type: 'url' }]"
-				>
-					<a-input v-model:value="formState.url" />
-				</a-form-item>
-				<a-form-item
-					label="URL Hash"
-					name="url_hash"
-					:rules="[{ required: true }]"
-				>
-					<a-input v-model:value="formState.url_hash" />
-				</a-form-item>
-				<a-form-item
-					label="Proposal Account Address"
-					name="proposalAddress"
-					:rules="[
-						{
-							required: true,
-						},
-					]"
-				>
-					<a-input v-model:value="formState.proposalAddress" :disabled="true" />
-				</a-form-item>
-				<a-form-item
-					label="Voting Date"
-					name="vote_date"
-					:rules="[{ required: true }]"
-				>
-					<a-range-picker
-						format="YYYY-MM-DD HH:mm:ss"
-						value-format="YYYY-MM-DD HH:mm:ss"
-						:disabled-date="disabledDate"
-						:disabled-time="disabledRangeTime"
-						showTime
-						v-model:value="formState.vote_date"
+	<opt-in-error>
```

Comment:
Why are we changing Proposal.vue file for opt in button?

Reply:
Robert asked me to not display any form until the user has opted in dao app, thus all three form are displayed only when app is opted in